package.json

```diff
@@ -27,10 +27,12 @@
     "karma-phantomjs-launcher": "^1.0.0",
     "karma-webpack": "^1.7.0",
     "mocha": "^2.4.5",
+    "nightwatch": "^0.9.8",
     "node-sass": "^3.6.0",
     "phantomjs-prebuilt": "^2.1.7",
     "raw-loader": "^0.5.1",
     "sass-loader": "^3.2.0",
+    "selenium-binaries": "^0.8.0",
```

Comment:
I'd fix down the version number of selenium...its tends to change frequently and you will be constantly having the tweak the path to the binary ( see note above ).